primefaces/src/main/resources/META-INF/resources/primefaces/sticky/sticky.js

```diff
@@ -98,7 +96,7 @@ PrimeFaces.widget.Sticky = PrimeFaces.widget.BaseWidget.extend({
             this.ghost = $('<div class="ui-sticky-ghost"></div>').height(this.target.outerHeight()).insertBefore(this.target);
             this.target.width(this.ghost.outerWidth() - (this.target.outerWidth() - this.target.width()));
             this.fixed = true;
-            win.scrollTop(winScrollTop);
+            $(window).scrollTop(winScrollTop);
```

Comment:
for such cases, its actually not required but you did it for uniformity?

Reply:
Correct I just did it for uniformity.